app/controllers/concerns/ab_testable.rb

```diff
@@ -5,6 +5,10 @@ module AbTestable
     helper_method :ab_variant_for, :current_ab_variants
   end
 
+  def show_test_variant(name_of_test, variant_name)
+    Rails.env.production? && ab_variant_for(name_of_test) == variant_name
```

Comment:
Why do we need to test for prod here?

Reply:
Also the method name kinda implies to me that this has a side effect, plus we use just `test` (without name) in other places in this file and elsewhere – how about `current_variant?(test, variant)` instead?

Reply:
should more acurately be !Rails.env.test

Reply:
Do some of the tests expect the default variant and break if another one is selected?

I think a cleaner way in that case would be to configure the AB test to always show the default variant in test in `config/ab_tests.yml`:

```yaml
shared:
  # ...
test:
  # Force test environment to always show default cookie banner
  2021_02_cookie_consent_test:
    none: 0
    default: 1
    top: 0
    bottom: 0
    modal: 0
    gds: 0
```